### DISCUSSION TITLE
Klick'r 3.0.0-beta03

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#426] [Announcements]
> [!IMPORTANT]
> **This is a beta build.**
> It might be unstable so please backup your scenarios before updating to this version. It will be only availble on direct download here, and not on PlayStore/FDroid.

## Bug fixes

* Fix a crash when updating from 2.4.2 with certain scenarios
* Fix a possible crash on Xiami devices
* Fix a crash on screen recording start on Samsung devices with Android 14
* Improve detection stability

**Full Changelog**: https://github.com/Nain57/Smart-AutoClicker/compare/3.0.0-beta02...3.0.0-beta03

<hr /><em>This discussion was created from the release <a href='https://github.com/Nain57/Smart-AutoClicker/releases/tag/3.0.0-beta03'>Klick'r 3.0.0-beta03</a>.</em>

**Comment by dginovker** [comment]
Wow, surprised you're still working on this! I first installed this app years ago. I remember just wanting a basic autoclicker and finding it. I don't even remember if Image Recognition clicking was a thing back then. But this autoclicker has served me very well ^^. Many P2W games were beat without needing to pay. Many grinds were reduced to simple dopamine hits from checking the phone. The project is awesome, and so are you! ~~
